### PR TITLE
Batch fix doc issues in tiup docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Batch fix typos in tiup docs

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/4868 https://github.com/pingcap/docs/pull/5292
 
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-command-mirror-genkey.md [hunk 1]
```diff
@@ -18,25 +18,30 @@ title: tiup mirror genkey
...
 ## 语法
 
-```sh
+```shell
 tiup mirror genkey [flags]
 ```
 
 ## 选项
 
 ### -n, --name（string，默认 private）
```
Comment (TomShawn) [inline tiup/tiup-command-mirror-genkey.md:27]:
```suggestion
### -n, --name
```

File: tiup/tiup-command-list.md [hunk 2]
```diff
@@ -8,30 +8,37 @@ title: tiup list
...
 
 ## 选项
 
-### --all（boolean，默认 false）
+### --all
 
-显示所有组件。默认只显示非隐藏组件。
+- 显示所有组件。默认只显示非隐藏组件。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
```
Comment (TomShawn) [inline tiup/tiup-command-list.md:24]:
```suggestion
```

File: tiup/tiup-component-cluster-clean.md [hunk 3]
```diff
@@ -6,34 +6,39 @@ title: tiup cluster clean
...
 
-### --log（boolean，默认 false）
+### --log
 
-该选项开启日志清理，若不指定该选项，也不指定 `--all`，则不清理日志。
+- 该选项开启日志清理，若不指定该选项，也不指定 `--all`，则不清理日志。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
 ### --ignore-node（StringArray，默认为空）
```
Comment (qiancai) [inline tiup/tiup-component-cluster-clean.md:43]: Should the info here about the data type and default value be updated to an unordered list too?

File: tiup/tiup-component-dm.md [hunk 4]
```diff
@@ -32,17 +32,23 @@ tiup dm [command] [flags]
...
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-### -v, --version（boolean，默认 false）
+### -v, --version
 
-输出 tiup-dm 当前版本信息。
+    - 输出 tiup-dm 当前版本信息。
+    - 数据类型：`BOOLEAN`
+    - 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
```
Comment (TomShawn) [inline tiup/tiup-component-dm.md:45]:
```suggestion
- 输出 tiup-dm 当前版本信息。
- 数据类型：`BOOLEAN`
- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
```

File: tiup/tiup-command-mirror-clone.md [hunk 5]
```diff
@@ -17,21 +17,29 @@ tiup mirror clone <target-dir> [global version] [flags]
...
-是否克隆整个镜像。指定该选项后会完整从目标镜像克隆所有组件的所有版本，此时其他指定的选项将失效。
+- 是否克隆整个镜像。指定该选项后会完整从目标镜像克隆所有组件的所有版本，此时其他指定的选项将失效。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-### -a, --arch（strings，默认 amd64,arm64）
+### -a, --arch
 
-仅克隆能在指定平台上运行的组件。
+- 仅克隆能在指定平台上运行的组件。
```
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-command-mirror-clone.md:28]: 可以指定

[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-command-mirror-clone.md:28]: 原来的类型是 strings，复数

File: tiup/tiup-command-mirror-clone.md [hunk 6]
```diff
@@ -17,21 +17,29 @@ tiup mirror clone <target-dir> [global version] [flags]
...
+- 数据类型：`STRING`
+- 如果未指定该选项，默认克隆 AMD64 和 ARM64 平台的组件 (`amd64,arm64`)。
 
-### -o, --os（strings，默认 linux,darwin）
+### -o, --os
 
-仅克隆能在指定操作系统上运行的组件。
+- 仅克隆能在指定操作系统上运行的组件。
```
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-command-mirror-clone.md:34]: 同上

File: tiup/tiup-command-mirror-modify.md [hunk 7]
```diff
@@ -23,24 +23,31 @@ tiup mirror modify <component>[:version] [flags]
...
 >
 > 该选项只能应用于组件上，无法应用于组件的版本上。
 
-### --standalone（boolean，默认 false）
+### --standalone
 
-该组件是否可独立运行。本参数目前尚未启用。
+- 该组件是否可独立运行。本参数目前尚未启用。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
```
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-command-mirror-modify.md:50]: 可以的

File: tiup/tiup-component-cluster-deploy.md [hunk 8]
```diff
@@ -26,29 +26,38 @@ tiup cluster deploy <cluster-name> <version> <topology.yaml> [flags]
...
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
 如果想要跳过该项检查，可以使用该选项。
```
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-component-cluster-deploy.md:41]: 也是可以的

File: tiup/tiup-component-cluster-clean.md (hunk 3 above)
Comment (TomShawn) [reply tiup/tiup-component-cluster-clean.md:43]: There are hundreds of similar modifications to make in the TiUP docs and the workload can be huge. This PR only updates/refines a part of them. Other unrefined titles will be refined by upcoming PRs.

File: tiup/tiup-component-cluster-destroy.md [hunk 9]
```diff
@@ -12,17 +12,19 @@ title: tiup cluster destroy
...
 
-### --force（boolean，默认 false）
+### --force
 
-在某些情况下，有可能集群中的某些节点已经宕机，导致无法通过 SSH 连接到节点进行操作，这个时候可以通过 `--force` 选项忽略这些错误。
+- 在某些情况下，有可能集群中的某些节点已经宕机，导致无法通过 SSH 连接到节点进行操作，这个时候可以通过 `--force` 选项忽略这些错误。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
 ### --retain-node-data（StringArray，默认为空）
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply tiup/tiup-component-cluster-destroy.md:29]: There are hundreds of similar modifications to make in the TiUP docs and the workload can be huge. This PR only updates/refines a part of them. Other unrefined titles will be refined in upcoming PRs.

**Comment by TomShawn** [comment]
@qiancai All your comments above are addressed in https://github.com/pingcap/docs-cn/pull/6042/commits/7a8067868ca3dc77574ed36bddf64049692c3441. PTAL again. Thanks!

File: tiup/tiup-component-cluster-deploy.md [hunk 10]
```diff
@@ -26,29 +26,37 @@ tiup cluster deploy <cluster-name> <version> <topology.yaml> [flags]
...
-在连接目标机器时使用密码登陆，不可和 `-i/--identity_file` 同时使用。
+- 在连接目标机器时使用密码登陆，不可和 `-i/--identity_file` 同时使用。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-deploy.md:37]:
```suggestion
- 在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```

File: tiup/tiup-component-cluster-deploy.md [hunk 11]
```diff
@@ -26,29 +26,37 @@ tiup cluster deploy <cluster-name> <version> <topology.yaml> [flags]
...
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-deploy.md:38]: 因为 “该项检查“ 针对的是第一个 item 里说的检查，为了内容连贯性，如果把 “如果想要跳过该项检查，可以使用该选项。”放到第一个 item 的结尾处可能更清楚些 ：）

File: tiup/tiup-component-cluster-reload.md [hunk 12]
```diff
@@ -28,11 +30,12 @@ tiup cluster reload <cluster-name> [flags]
 >
 > 若出现跳过等待直接重启的情况，服务性能可能会出现抖动。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
-
-如果想要跳过该项检查，可以使用该选项。
+- 在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-reload.md:35]:
```suggestion
- 在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```

File: tiup/tiup-component-cluster-reload.md [hunk 13]
```diff
@@ -28,11 +30,12 @@ tiup cluster reload <cluster-name> [flags]
...
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-reload.md:36]: ditto

File: tiup/tiup-component-cluster-upgrade.md [hunk 14]
```diff
@@ -33,19 +35,24 @@ tiup cluster upgrade <cluster-name> <version> [flags]
 > 
 > 若出现跳过等待直接升级的情况，服务性能可能会出现抖动。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在二进制文件更新之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 在二进制文件更新之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-upgrade.md:40]:
```suggestion
- 在二进制文件更新之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
```

File: tiup/tiup-component-cluster-upgrade.md [hunk 15]
```diff
@@ -33,19 +35,24 @@ tiup cluster upgrade <cluster-name> <version> [flags]
...
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-upgrade.md:41]: ditto

File: tiup/tiup-component-cluster-deploy.md (hunk 11 above)
Comment (TomShawn) [inline tiup/tiup-component-cluster-deploy.md]:
```suggestion
- 在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

File: tiup/tiup-component-cluster-reload.md (hunk 13 above)
Comment (TomShawn) [inline tiup/tiup-component-cluster-reload.md]:
```suggestion
在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

File: tiup/tiup-component-cluster-deploy.md [hunk 16]
```diff
@@ -26,29 +26,37 @@ tiup cluster deploy <cluster-name> <version> <topology.yaml> [flags]
...
+- 在连接目标机器时使用密码登陆，不可和 `-i/--identity_file` 同时使用。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-deploy.md:38]:
```suggestion
在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

File: tiup/tiup-component-cluster-reload.md [hunk 17]
```diff
@@ -28,11 +30,12 @@ tiup cluster reload <cluster-name> [flags]
...
 > 若出现跳过等待直接重启的情况，服务性能可能会出现抖动。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在组件二进制文件部署之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
-
-如果想要跳过该项检查，可以使用该选项。
+- 在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-reload.md:36]:
```suggestion
- 在组件二进制文件部署之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

File: tiup/tiup-component-cluster-upgrade.md (hunk 15 above)
Comment (TomShawn) [inline tiup/tiup-component-cluster-upgrade.md:41]:
```suggestion
```

File: tiup/tiup-component-cluster-upgrade.md [hunk 18]
```diff
@@ -33,20 +35,25 @@ tiup cluster upgrade <cluster-name> <version> [flags]
 > 
 > 若出现跳过等待直接升级的情况，服务性能可能会出现抖动。
 
-### --ignore-config-check（boolean，默认 false）
+### --ignore-config-check
 
-在二进制文件更新之后，会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<bianry>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 在二进制文件更新之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。
+- 如果想要跳过该项检查，可以使用该选项。
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-upgrade.md:41]:
```suggestion
- 在二进制文件更新之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

Comment (TomShawn) [inline tiup/tiup-component-cluster-upgrade.md]:
```suggestion
- 在二进制文件更新之后，TiUP 会对 TiDB，TiKV 和 PD 组件执行配置检查，检查方式为 `<binary> --config-check <config-file>`，其中 `<binary>` 为新部署的二进制文件的路径，`<config-file>` 为根据用户配置生成的配置文件。如果想要跳过该项检查，可以使用该选项。
```

**Review by qiancai — APPROVED** [review]

**Review by CharLotteiu — APPROVED** [review]

**Pull request merged**